### PR TITLE
Collect Optional Email from Applicant

### PR DESCRIPTION
**Pull request by RonanCog**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by SangeethaPonnusamy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Running the application requires the following tools to be installed in your env
 - [yarn](https://yarnpkg.com/)
 - [Docker](https://www.docker.com)
 
-### Running the application
+#### Running the application
```
Comment (yogendraU) [inline README.md:14]: Is this file change required?

File: README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ $ yarn start
 
 The applications's home page will be available at https://localhost:4000
 
-### Running with Docker
+## Running with Docker
```
Comment (yogendraU) [inline README.md:35]: Is this change required?

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Murshadhaque) [reply README.md]: reverted

File: README.md (hunk 1 above)
Comment (Murshadhaque) [reply README.md]: reverted

**Review by yogendraU — APPROVED** [review]

**Review by hmcts-kodatala — APPROVED** [review]
Ok

**Review by saravanakumar-palanichamy — APPROVED** [review]

**Pull request merged**